Jenkinsfile

```
@@ -249,7 +249,8 @@ String functional_packages(String distro) {
                   "hdf5-vol-daos-openmpi3-tests-daos-1 " +
                   "MACSio-mpich " +
                   "MACSio-openmpi3 " +
-                  "mpifileutils-mpich-daos-1 "
+                  "mpifileutils-mpich-daos-1 " +
+                  "maven "
```

This should be a `Requires: maven` in the `daos-tests-java` sub-package.
```suggestion
                  "mpifileutils-mpich-daos-1 "
```